docs/source/policies/security_response_policy.rst

```diff
@@ -0,0 +1,28 @@
+NI LinuxRT Security Response Policy
+===================================
+
+NI is committed to resolving major security vulnerabilities discovered in the x64 NI LinuxRT distribution images and core package feeds.
+
+The LinuxRT development team regularly monitors the `NIST National Vulnerability Database <https://nvd.nist.gov/>`_ for published vulnerabilities, and uses that information to audit the x64 NILRT distribution software. When a vulnerability is discovered, the team evaluates its impact to our products and our customers' supported workflows, and assigns the vulnerability a vendor severity score. In many cases, NI's vendor score agrees with the NIST-assigned score. But the team may downgrade a vulnerability if it does not affect the system in a supported configuration.
+
+Vulnerabilities which the team assesses as being high-severity are patched or otherwise fixed in the next release of the distribution. Critical-severity vulnerabilities may have their fixes backported to the latest distribution release, if it is similarly affected.
+
+To ensure that your NI LinuxRT deployment remains free of vulnerabilities, it is best to schedule regular upgrades of your deployment's firmware and base system images.
+
+
+Reporting Security Issues
+-------------------------
+
+We encourage you to report security vulnerabilities to us privately so that we can follow a coordinated disclosure process, allowing us time to thoroughly investigate security issues and publicly disclose them when appropriate.
```

Comment:
May be useful to link to an overall NI security page if we have one. Just chopping off part of the PGP URL below I found [this page][1].

[1]: <https://www.ni.com/en/support/security.html>

Reply:
Sure will fix.

Reply:
Fixed Patch V2.